corehq/motech/generic_inbound/templates/generic_inbound/request_log_detail.html

```diff
@@ -4,6 +4,15 @@
 
 {% block page_content %}
 
+{% captureas status_css %}
+  {% case log.status 'success' %} success
+  {% case 'filtered' %} default
+  {% case 'reverted' %} warning
+  {% case 'error' 'validation-failed' %} danger
+  {% else %} default
+  {% endcase %}
+{% endcaptureas %}
```

Comment:
random note - I started this trying to pass the log objects as-is into the template, so we can handle all the display logic in the template language, but this felt like a case where it might've been more straightforward to do a serialization/enrichment step in the view code, so this could be defined in python.  Managed to get it working, but happy to revisit that decision if this feels too unorthodox or if other use cases pop up that don't fit the current model.

Reply:
I didn't know about 'captureas' 🎓 

This seems fine to me